### PR TITLE
Code decaffeination

### PR DESCRIPTION
**Pull request by mserranom**
Module decaffeination.

### CODE REVIEW

**Comment by mserranom** [comment]
Not sure what to do with adding linting to CI etc. From the Makefile, it's supposed to remain untouched?

```
# This file was auto-generated, do not edit it directly.
# Instead run bin/update_build_scripts from
# https://github.com/sharelatex/sharelatex-dev-environment
# Version: 1.1.12
```

**Comment by das7pad** [comment]
@mserranom You could change the project type in the buildscript.txt file to `es` and rerun the script you just quoted. This should add the lint tasks into the package.json, Makefile and Jenkinsfile.

Your merge commit from master dropped the eslint packages in the `npm-shrinkwrap.json` which you added previously in this PR, maybe add them back?

File: app.js [hunk 1]
```diff
@@ -35,16 +35,6 @@ server.get('/status', (req, res) => res.send({ status: 'spelling api is up' }))
 
 server.get('/health_check', HealthCheckController.healthCheck)
 
-const profiler = require('v8-profiler')
```
Comment (das7pad) [inline app.js]: See #20 for a replacement profiler.

**Comment by mserranom** [comment]
closing in favour of https://github.com/overleaf/spelling/pull/24

**Pull request closed without merging**